gossipd/gossip.c

```diff
@@ -1860,10 +1860,6 @@ static void gossip_disable_outgoing_halfchan(struct daemon *daemon,
 		    "Unable to parse previously accepted channel_update");
 	}
 
-	timestamp = time_now().ts.tv_sec;
-	if (timestamp <= hc->last_timestamp)
-		timestamp = hc->last_timestamp + 1;
-
```

Comment:
Yes, left over from 57794b928552d08f8b73a9612ada61b485abc9b0 where this function was reworked.